Tests/AutoCollection/HttpDependencyParser.tests.ts

```diff
@@ -56,8 +56,8 @@ describe("AutoCollection/HttpDependencyParser", () => {
             assert.equal(dependencyTelemetry.dependencyTypeName, Contracts.RemoteDependencyDataConstants.TYPE_HTTP);
             assert.equal(dependencyTelemetry.success, true);
             assert.equal(dependencyTelemetry.name, "GET /search");
-            assert.equal(dependencyTelemetry.data, "https://a.bing.com:443/search");
-            assert.equal(dependencyTelemetry.target, "a.bing.com:443");
+            assert.equal(dependencyTelemetry.data, "https://a.bing.com/search");
+            assert.equal(dependencyTelemetry.target, "a.bing.com");
```

Comment:
Remove port because : 
When using URL API to create a new object, when port matches with the protocol, the port is dropped. 

For example: 
- new URL("**https**://a.bing.com:443/search").port returns ""      - port dropped since https matches with 443
- new URL("**http**://a.bing.com:443/search").port returns "443"  - port not dropped
- new URL("**http**://a.bing.com:80/search").port returns ""          - port dropped since http matches with 80
- new URL("**https**://a.bing.com:80/search").port returns "80"     - port not dropped
- new URL("**https**://a.bing.com:1234/search").port returns "1234"   - port not dropped
- new URL("**http**://a.bing.com:1234/search").port returns "1234"     - port not dropped 

![image](https://user-images.githubusercontent.com/20292261/126006027-d1b16635-ef43-4596-a0ee-d28ba1511ef0.png)

Confirmed with Nev that : the default port for a protocol is often never sent @hectorhdzg can you confirm this? Thanks.